purchases/src/main/kotlin/com/revenuecat/purchases/common/diagnostics/DiagnosticsEntry.kt

```diff
@@ -39,7 +39,7 @@ sealed class DiagnosticsEntry(val diagnosticType: String) {
         }
     }
 
-    data class Counter(
+    internal data class Counter(
```

Comment:
Hmm is it necessary to make the elements of an internal sealed class also internal? And if so, should we also make the `Event` internal?

Reply:
Oh no, it's not necessary, we can't revert this change :smile: